formapp/templates/controlpanel.html

```diff
@@ -303,21 +330,30 @@ <h3>API</h3>
   <div id="tabs-5">
     <h2>Lock Formbar</h2>
     Only whitelisted users will be able to make changes.<br>
-    <input type="checkbox" id="lock" onchange="request.open('GET', '/controlpanel?locked=' + this.checked); request.send();"><label for="lock">Lock</label><br>
+    <input type="checkbox" id="lock"
+      onchange="request.open('GET', '/controlpanel?locked=' + this.checked); request.send();"><label
+      for="lock">Lock</label><br>
```

Comment:
Manually break the line between tags. The formatter is splitting them halfway through the tag.